SimpleWAWS/Global.asax.cs

```diff
@@ -151,17 +162,19 @@ protected void Application_AuthenticateRequest(Object sender, EventArgs e)
 
             if (!SecurityManager.TryAuthenticateSessionCookie(context))
             {
+                //
+
                 // Support requests from non-browsers with bearer headers
-                if (context.IsFunctionsPortalRequest() &&
-                    !context.IsBrowserRequest() &&
+                if (context.IsFunctionsPortalRequest() && !context.IsBrowserRequest() &&
```

Comment:
can we rename the `IsFunctionsPortalRequest` to `IsFunctionsPortalBackendRequest`